.github/pull_request_template.md

```diff
@@ -16,12 +16,12 @@ Fixes #ISSUE-NUMBER (eg. Fixes #147)
 
 Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
 
-- [ ] Test A
-- [ ] Test B
+- Test A
+- Test B
 
 
 #### Checklist:
 
-- [ ] My code follows the style guidelines of this project and performed a self-review of my own code
-- [ ] New and existing tests pass locally with my changes
-- [ ] I have made corresponding changes to the documentation (If Any)
+- [  ] My code follows the style guidelines of this project and performed a self-review of my own code
```

Comment:
I'm worried people might get these wrong and will not check/uncheck/correct these.  
For the checklist I would be ok to leave these as tasks. We can use these to remind people to check these.
@bumi what do you think?

Reply:
yeah, the problem is with the last one! documentation not necessary for every pr other two is a must. so if we keep as it is we will always have ```2 out of 3 tasks done

Reply:
Hm, maybe we can change it to something like "I checked if I need to made corresponding changes to the documentation (and made these if needed)" so it can be checked also if no changes need to be done? What do you think?

Reply:
yep, good idea maybe we can indicate that comments/docs should be checked updated or written?

Reply:
and maybe we should add something non code releated:

```
- [ ] I have made a somersault
- [ ] I've been excellent today
```

:D

Reply:
- [ ] I like 🐢🐢🐢

Reply:
- [X] My code follows the style guidelines of this project and performed a self-review of my own code
- [X] New and existing tests pass locally with my changes
> I checked if I need to made corresponding changes to the documentation (and made these if needed)

Reply:
what do you think

Reply:
I would prefer a checkbox to keep the same style.  Everything else might be confusing for people.

Reply:
> I would prefer a checkbox to keep the same style. Everything else might be confusing for people.

yeah true! so single spaces in brackets with the last statement changed!

- [X] My code follows the style guidelines of this project and performed a self-review of my own code
- [X] New and existing tests pass locally with my changes
- [X] I checked if I need to make corresponding changes to the documentation (and made these if needed)

Reply:
also if all three are now must what if we keep them already checked! will it be effective?

Reply:
did the suggested changes!